fields/default/fields_list.json

```diff
@@ -404,6 +404,17 @@
             "base.ext"
         ]
     },
+    {
+        "path": "additional_info.vehicle.owner.phone_number",
+        "description": "Дополнительная информация: Номер телефона владельца",
+        "types": [
+            "string"
+        ],
+        "fillable_by": [
+            "base",
+            "base.ext"
+        ]
+    },
     {
         "path": "additional_info.vehicle.owner.geo",
```

Comment:
We're not decribe intenal fields for `geo` object. How I must decribe field `additional_info.vehicle.owner.geo.postal_code` in `fields_list.json`?

Reply:
No, we shouldn't